src/c3chart/CommonND.js

```diff
@@ -133,6 +135,23 @@
             break;
         }
 
+        if(this.y2() instanceof Array && this.y2().length > 0){
+            var axesObj = {};
+            var y2arr = this.y2();
+            this._config.axis.y2 = {
+                show: y2arr.length > 0
+            };
+            if(this.columns().length > 1){
+                var cols = this.columns();
+                for(var i = 0;i<y2arr.length;i++){
+                    for(var j = 1;j<cols.length;j++){
+                        axesObj[cols[j]] = y2arr.indexOf(cols[j]) !== -1 ? "y2" : "y";
+                    }
+                }
+                this._config.data.axes = axesObj;
+            }
+        }
```

Comment:
@dtsnell4 It's an issue with the chart. As far as I'm aware... C3 doesn't allow adding a second Y axis to a chart that's already initialized.